colossalai/inference/README.md

```diff
@@ -53,27 +64,39 @@ pip install -e .
 
 ### Requirements
 
-dependencies
+Install dependencies.
 
 ```bash
-pytorch= 1.13.1 (gpu)
-cuda>= 11.6
-transformers= 4.30.2
-triton
-# for install flash-attention
-flash-attention
+pip install -r requirements.txt
+pip install transformers==4.34.0
 
 # install lightllm since we depend on lightllm triton kernels
-git clone https://github.com/ModelTC/lightllm 
+git clone https://github.com/ModelTC/lightllm
```

Comment:
This can also be moved into requirements.txt, see https://stackoverflow.com/questions/13685920/install-specific-git-commit-with-pip

Reply:
Same.